sync/class.jetpack-sync-module-users.php

```diff
@@ -55,7 +55,6 @@ public function init_listeners( $callable ) {
 		add_action( 'jetpack_wp_login', $callable, 10, 3 );
 
 		add_action( 'wp_logout', $callable, 10, 0 );
-		add_action( 'wp_masterbar_logout', $callable, 10, 0 );
```

Comment:
No need to sync this anymore since there is log out option in the Masterbar anymore.